front-end/studio/src/app/pages/apis/{apiId}/_components/generate-project.wizard.html

```diff
@@ -232,6 +232,15 @@ <h3 class="blank-slate-pf-main-action">Loading Wizard Data</h3>
                                                 <form-error-message [inputModel]="javaPackageInput" [type]="'pattern'">Please enter a valid Java package name.</form-error-message>
                                             </div>
                                         </div>
+                                        <div class="form-group"> 
+                                            <label class="col-sm-2 control-label" for="springboot_templateRepo">Templates Repository</label>
+                                             <div class="col-sm-10">
+                                                 <input #templateRepoInput="ngModel" name="templateRepo" type="text" 
+                                                        id="springboot_templateRepo" class="form-control" placeholder="https://github.com/Djai-Studio.git"
```

Comment:
@AndreiZiatiev  please change the placeholder value to https://github.com/link-to-your-template-repo.git

Reply:
+